awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx

```diff
@@ -209,7 +210,14 @@ function JobDetail({ job, i18n }) {
             value={
               <ChipGroup numChips={5}>
                 {credentials.map(c => (
-                  <CredentialChip key={c.id} credential={c} isReadOnly />
+                  <div css="padding-right: 5px" key={c.id}>
```

Comment:
same here...since you are using this in a few places, it might be worth defining it in a components file.